assets/javascripts/discourse/controllers/group-reports-show.js

```diff
@@ -84,7 +92,7 @@ export default class GroupReportsShowController extends Controller {
         }),
       {
         onAfterSave: (savedData) => {
-          const bookmark = Bookmark.create(savedData);
+          const bookmark = this.store.createRecord("bookmark", savedData);
```

Comment:
Do I need to save the record?

Reply:
Those two forms should be pretty much equivalent. So if no extra save was required before, it shouldn't be required now.